packages/api-client/src/api/updateCurrentUser/index.ts

```diff
@@ -0,0 +1,16 @@
+import { ApiContext } from '../../types';
+import getCurrentBearerToken from '../authentication/getCurrentBearerToken';
+
+export default async function updateUser({ client, config }: ApiContext, { email, firstName, lastName }): Promise<void> {
+  const bearerToken = await getCurrentBearerToken({ client, config });
+  const updateData = {
+    user: {
+      email,
+      first_name: firstName,
+      last_name: lastName}
```

Comment:
This formatting feels a bit odd (curly braces at the end of the line)- could you take a look?